python/libcudf/libcudf/_version.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
```suggestion
# Copyright (c) 2024, NVIDIA CORPORATION.
```

Reply:
re-opening this thread.

The `verify-copyright` hook really wants this to be 2023, despite being a new file and the earliest commit on this PR being from 2024 (https://github.com/rapidsai/cudf/pull/15483/commits/a3de193f2a89e646a7f6e14cfa6d3f56eab7346c).

Confirmed that `cudf` is using the latest version of the hook.

https://github.com/rapidsai/cudf/blob/555734dee7a8fb10f50c8609a8e4fb2c025e6305/.pre-commit-config.yaml#L145-L149

Will look into it (after I get other things working here and ready for review).

Reply:
(I am still investigating this, but let's not block the PR on it...if this gets merged without this fixed, I'll put up an issue and look into it after)

Reply:
I agree that this is not a blocker and can be pursued later.

Reply:
Opened https://github.com/rapidsai/pre-commit-hooks/issues/48 to track this

I put this there instead of `cudf` because I strongly suspect it's an issue in the `verify-copyright` hook.